packages/core/src/internal/connection-holder.ts

```diff
@@ -161,12 +163,16 @@ class ConnectionHolder implements ConnectionHolderInterface {
   }
 
   private async _createConnectionPromise (connectionProvider: ConnectionProvider): Promise<Connection | null> {
+    this._connectionError = undefined
```

Comment:
you do not need to reset it here, do you?

Reply:
I removed the changes in the connection-holder, to avoid effects in other areas.